src/api/config.ts

```diff
@@ -1,12 +1,19 @@
 import express from 'express';
 import { Config } from '../entities/config';
 import logger from '../logger';
+import { KnownConfig } from '../types/config';
 
 export const config = express.Router();
 
 config.get('/', async (req, res) => {
   try {
-    res.send(await Config.find({ order: { key: 'ASC' } }));
+    const configItems: Config[] = await Config.find();
+    Object.values(KnownConfig).forEach((knownConfigItem): void => {
```

Comment:
If we're not using the keys of the enum, let's just make it an array at the top of this file and call it `knownConfigKeys`

Reply:
Never mind, enum has other purposes